packages/airnode-validator/src/validator.test.ts

```diff
@@ -3,13 +3,13 @@ import fs from 'fs';
 import * as validator from './validator';
 import { extraFieldMessage, formattingMessage, missingParamMessage } from './utils/messages';
 import { getPath } from './commands/utils';
-import { error } from './utils/logger';
+import { error, warn } from './utils/logger';
 import { Log } from './types';
 
 const messages: Log[] = [];
 
-const endpointsPath = getPath('endpoints.json', messages, 'pre-alpha');
-const oisPath = getPath('ois.json', messages, 'pre-alpha');
+const endpointsPath = getPath('endpoints.json', messages, '1.0');
```

Comment:
How will this look like when we release a new version of OIS? Is there a ticket for it?

Reply:
I don't think there's any plan for how to deal with this as of now. And I don't know about the ticket either.

Reply:
Tracked in https://api3dao.atlassian.net/browse/AN-465